pkg/controller/job/controller_test.go

```diff
@@ -161,7 +161,7 @@ func TestControllerSyncJob(t *testing.T) {
 		"too few active pods, with controller error": {
 			2, 5,
 			fmt.Errorf("Fake error"), 0, 1, 1, 0,
-			0, 0, 1, 1, 0, false,
+			1, 0, 1, 1, 0, false,
```

Comment:
The test functions didn't take failed patch into account. I need to know the controller has sent failed requests, thus the change.